### DISCUSSION TITLE
Where is the database?

### DISCUSSION THREAD

**Discussion by bachkov** [AppFlowy-IO/AppFlowy#455] [General]
Great tool guys! Can you tell me where I can find my notes saved?

A little feedback:
I was looking for alternative to notepad, with dark theme, that was stylish and minimalistic, as well as being very lightweight, as I'm running heavy 3d and compositing software, and having to open things like Evernote, Sublime, Word while rendering, to take down notes is not the best option imo, and notepad, didn't have bullet points, even more cool - tasks, and grouping of lists. Of course, I saw some people didn't like the design approach - for me personally, if you're going into the direction of creating more feature rich app - people like me would have to stay with the lower version of your software, unless you're able to keep the same optimization. Or another option maybe - separate the software into two branches - one "lite" app that is blazing fast and keeps things quite simple and one "full" app that may take a couple of seconds to load.

Imo you were able to hit a sweet spot in complexity and lite-nesnes :D - don't lose it please :) 

Btw - if you're going mobile - it would be cool if you create a widget that loads a list and I can give you an example of widget I use, that is simple yet poweful imo design to change color /opacity/size of the elements ...although that may conflict with the colors set in the list that will be shown, anyway, attaching screenshot: (app is 4note i think):

![Capture](https://user-images.githubusercontent.com/41993314/163570806-ef112bdb-cfbf-4a31-938f-b03bb9fa793b.PNG)

**Comment by annieappflowy** [comment]
@bachkov , very much appreciate your feedback. It's super valuable to us. We will keep in mind with this input as we progress
> Or another option maybe - separate the software into two branches - one "lite" app that is blazing fast and keeps things quite simple and one "full" app that may take a couple of seconds to load.

RE: mobile list widget, could you please provide me with an example? Is it a bulleted list? Or it's a list database view?

Thanks,
Annie

**Reply by bachkov** [reply]
Here is the widget I was talking about. In it, you enter just plain text and it displays it on the screen, but my idea was instead to just chose some of the pages from AppFlowy and display it with some options of chosing size, color and opacity. This widget was called 4note, it's on google play, but yep, with more options with checkboxes and bullets, etc, implementation of page from AppFlowy, would be much more complex I guess.
<img src="https://user-images.githubusercontent.com/41993314/164886352-9db1e977-d197-444d-8ee3-918a2bbbb818.jpg" width="250" height="auto"><img src="https://user-images.githubusercontent.com/41993314/164886341-5051d322-b2f5-4b59-96a3-09f9590173bf.jpg" width="250" height="auto">
<img src="https://user-images.githubusercontent.com/41993314/164886355-59eab21e-4862-45e5-ada7-800ae6ac3aa4.jpg" width="250" height="auto"><img src="https://user-images.githubusercontent.com/41993314/164886348-bc8a67ca-d08a-45ab-b498-31619199fe4c.jpg" width="250" height="auto">

**Comment by annieappflowy** [comment]
@bachkov the data is stored on your computer. You have to give us more details about your device in order for us to locate the path.

**Reply by bachkov** [reply]
I'm on Windows 10, I've put AppFlowy in this directory C:\Portable\AppFlowy, I'm just interested how to backup the database when I have to reinstall the computer.
Thank you! :)

**Comment by andercard0** [comment]
Hi, where is the database file to backup please ? read this discussion but is not clear to me where the db with my notes are located

I.E in massCode i can indicate where my data/db should be: 

![image](https://user-images.githubusercontent.com/43047877/191633573-7eb29af1-9156-45b9-abdb-26959f52b9a5.png)

**Reply by annieappflowy** [reply]
Did you build it from source? I can help you locate the data folder, which can be backed up to somewhere else.

**Reply by andercard0** [reply]
> Did you build it from source? I can help you locate the data folder, which can be backed up to somewhere else.

Hi, no I didn't. only make use of the assets available here in github. So, if the data folder is where my db relies on, it is safe to say that i have a backup ? 🤔
